### PR TITLE
Escaping HTML strings

### PR DESCRIPTION
**Pull request by octavianCraciun**

### CODE REVIEW

File: jquery.flot.axislabels.js [hunk 1]
```diff
@@ -159,9 +174,26 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
     };
 
     HtmlAxisLabel.prototype.draw = function (box) {
+        var deentityify = function (string) {
```
Comment (cipix2000) [inline jquery.flot.axislabels.js]: is this even a word ? 

I would prefer something like toSafeHTMLString ....

File: jquery.flot.axislabels.js [hunk 2]
```diff
@@ -136,8 +134,25 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
...
+            var entityMap = {
+                '&': '&amp;',
+                '<': '&lt;',
+                '>': '&gt;',
+                '"': '&quot;',
+                "'": '&#39;',
+                '/': '&#x2F;',
+                '`': '&#x60;',
+                '=': '&#x3D;'
```
Comment (cipix2000) [inline jquery.flot.axislabels.js]: is = unsafe ? How ?

Comment (octavianCraciun) [inline jquery.flot.axislabels.js]: it's safe.

File: jquery.flot.axislabels.js (hunk 1 above)
Comment (octavianCraciun) [inline jquery.flot.axislabels.js]: Crockford's deentityify method - JavaScript: The Good Parts page 41

Comment (cipix2000) [inline jquery.flot.axislabels.js]: ok

File: jquery.flot.axislabels.js [hunk 3]
```diff
@@ -267,12 +299,29 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
...
+            };
+
+            return String(string).replace(/[&<>"'`=\/]/g, function fromEntityMap (s) {
+                return entityMap[s];
+            }); 
+        }
+
```
Comment (cristiingineru) [inline jquery.flot.axislabels.js]: Reuse the deentityify function.

**Pull request closed without merging**